packages/directlinespeech/src/createAdapters.js

```diff
@@ -1,19 +1,22 @@
-/* eslint complexity: ["error", 30] */
+/* eslint complexity: ["error", 33] */
 
 import { AudioConfig } from 'microsoft-cognitiveservices-speech-sdk/distrib/lib/src/sdk/Audio/AudioConfig';
 import { BotFrameworkConfig, DialogServiceConnector, PropertyId } from 'microsoft-cognitiveservices-speech-sdk';
 
 import createWebSpeechPonyfillFactory from './createWebSpeechPonyfillFactory';
 import DirectLineSpeech from './DirectLineSpeech';
 import patchDialogServiceConnectorInline from './patchDialogServiceConnectorInline';
+import refreshDirectLineToken from './utils/refreshDirectLineToken';
 import resolveFunctionOrReturnValue from './resolveFunctionOrReturnValue';
 
+const DIRECTLINE_TOKEN_RENEWAL_INTERVAL = 900000; // 15 minutes
 const TOKEN_RENEWAL_INTERVAL = 120000;
 
 export default async function create({
   audioConfig,
   audioContext,
   audioInputDeviceId,
+  enableInternalHTTPSupport = false,
```

Comment:
Inherently fasly if value is `undefined`

```suggestion
  enableInternalHTTPSupport,
```